go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/kubernetes-sigs/aws-alb-ingress-controller
 require (
 	github.com/appscode/jsonpatch v0.0.0-20190108182946-7c0e3b262f30 // indirect
 	github.com/aws/aws-k8s-tester/e2e/tester v0.0.0-20190907061006-260b0e114d90
-	github.com/aws/aws-sdk-go v1.23.21
+	github.com/aws/aws-sdk-go v1.25.43
```

Comment:
What in v1.25.43 is it that we need?

Reply:
the weighted-routing API(unfortunately, imds v2 is added in same revision). I updated it to a latest version.